tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -106,6 +110,17 @@ describe 'Annotator.Plugin.EnhancedAnchoring', ->
       assert.ok hl
       assert.equal hl.page, page
 
+    it 'announces the creation of the highlights in an event', ->
+      am = createAnchoringManager()
+
+      assert am.annotator.publish.notCalled
+
+      ann = createTestAnnotation "a1"
+      anchor = am.createAnchor(ann, ann.target[0]).result
+
+      hl = anchor.highlight[anchor.startPage]
+      assert am.annotator.publish.calledWith "highlightsCreated", [hl]
```

Comment:
Sinon provides it's own [assertions](http://sinonjs.org/docs/#assertions) for spies that give more detailed output in the error messages. We usually mix them into the chai `assert` object at the top of the test file.

```coffee
assert = chai.assert
sinon.assert.expose(assert, prefix: '')
```

They can then be used anywhere in the test file, for example:

```coffee
assert.calledWith(am.annotator.publish, 'highlightsCreated', [hl])
```